user_guide_src/source/libraries/security.rst

```diff
@@ -129,6 +129,10 @@ It is also possible to enable the CSRF filter only for specific methods::
         'post' => ['csrf'],
     ];
 
+.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`.
+    Because auto-routing permits any HTTP method to access a controller.
```

Comment:
```suggestion
.. Warning:: If you use ``$methods`` filters, you should :ref:`disable auto-routing <use-defined-routes-only>`
    because auto-routing permits any HTTP method to access a controller.
```